src/pages/market/index.jsx

```
@@ -1,39 +1,75 @@
+import { SORT_OPTS } from "@/constants/martket";
 
-export default function MarketPage() {
-  const API_URL = process.env.NEXT_PUBLIC_API_URL || 'http://localhost:8000/api/v1';
-  const { isPending, error, data, _isFetching } = useQuery({
+
+async function getCards(query) {
+  try {
+    const response = await fetch(process.env.NEXT_PUBLIC_API_URL + '/shop/cards?' + SORT_OPTS.get(query.sortOptionKey).value);
+    return response.json();
+  } catch (error) {
+    throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)